CHANGELOG.md

```diff
@@ -20,8 +20,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * Bounding box cropping for Aircraft and CUB200.
 * Pretrained weights for vision models with: `l2l.vision.models.get_pretrained_backbone()`.
+* New vision example: MAML++. (@[DubiousCactus](https://github.com/DubiousCactus))
 * Add `keep_requires_grad` flag to `detach_module`. ([Zhaofeng Wu](https://github.com/ZhaofengWu))
 
+### Changed
+
+* Implement LSLR (MAML++) in the MAML module. (@[DubiousCactus](https://github.com/DubiousCactus))
+
```

Comment:
Move to Unreleased.

Reply:
I removed the LSLR implementation since it's not relevant anymore.